Source/Meadow.Contracts/Hardware/Contracts/IApp.cs

```diff
@@ -8,6 +8,8 @@
     /// </summary>
     public interface IApp
     {
+        void InvokeOnMainThread(Action<object> action, object? state = null);
```

Comment:
should this be on the App? Or in MeadowOS?

I can't remember where they put it in Xamarin.

Reply:
Looks like Xamarin uses a static `MainThread.BeginInvokeOnMainThread()`. It's the App's main thread, so this feels right.